### PR TITLE
WIP #1538 remove relative expression just to get the CI running

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/PKSim.Core/Commands/NormalizeRelativeExpressionCommand.cs [hunk 1]
```diff
@@ -57,7 +57,9 @@ protected override void PerformExecuteWith(IExecutionContext context)
          var allRelExpressionContainer = _molecule.AllExpressionsContainers();
          if (!allRelExpressionContainer.Any()) return;
          var max = allRelExpressionContainer.Select(x => x.RelativeExpression).Max();
-         allRelExpressionContainer.Each(relExp => relExp.RelativeExpressionNorm = max == 0 ? 0 : relExp.RelativeExpression / max);
-      }
+
+         //TODO ZTMSE
```
Comment (msevestre) [inline src/PKSim.Core/Commands/NormalizeRelativeExpressionCommand.cs:61]: @Yuri05 I have a bunch of TODOs everywhere that I'll start working on as I move through the tests.

File: src/Db/Diff/PKSimDB_diff.txt [hunk 2]
```diff
@@ -1,2 +1,67 @@
-UPDATE tab_calculation_method_rates SET formula='(Q*(1-HCT))*C_pls' WHERE calculation_method='MassTransferLink_forTwoPores' AND formula_rate='PARAM_MassTransferIntestineToMucosaPlasma';
-DELETE FROM tab_rate_generic_parameters WHERE calculation_method='MassTransferLink_forTwoPores' AND formula_rate='PARAM_MassTransferIntestineToMucosaPlasma' AND path_id=1524 AND parameter_name='Lymph flow rate';
+DELETE FROM tab_calculation_method_rates WHERE calculation_method='ActiveProcess_PKSim' AND formula_rate='RelExpBloodCellsNormGlobal';
```
Comment (msevestre) [inline src/Db/Diff/PKSimDB_diff.txt:1]: @Yuri05 
I removed all parameters, rate etc.. related to norm parameters
In other formula, removed the usage of norm parameters to use the non norm parameters automatically

File: src/PKSim.Core/CoreConstants.cs [hunk 3]
```diff
@@ -837,13 +834,10 @@ public static class Parameters
...
          public static readonly string REFERENCE_CONCENTRATION = "Reference concentration";
          public static readonly string REL_EXP = "Relative expression";
-         public static readonly string NORM_SUFFIX = " (normalized)";
          public static readonly string REL_EXP_BLOOD_CELL = "Relative expression in blood cells";
-         public static readonly string REL_EXP_BLOOD_CELL_NORM = NormParameterFor(REL_EXP_BLOOD_CELL);
          public static readonly string REL_EXP_PLASMA = "Relative expression in plasma";
-         public static readonly string REL_EXP_PLASMA_NORM = NormParameterFor(REL_EXP_PLASMA);
          public static readonly string REL_EXP_VASC_ENDO = "Relative expression in vascular endothelium";
-         public static readonly string REL_EXP_VASC_ENDO_NORM = NormParameterFor(REL_EXP_VASC_ENDO);
+         public static readonly string NORM_SUFFIX = " (normalized)";
```
Comment (Yuri05) [inline src/PKSim.Core/CoreConstants.cs:840]: NORM_SUFFIX probably not required anymore?

Comment (msevestre) [reply src/PKSim.Core/CoreConstants.cs:840]: yes. I'll have to move it to the conversion project. Right now it's still used.
I am wondering if I may have to keep it when using an OLD Individual.... or an OLD Simulation to filter those parameters out. Or we'll have to update a bunch of stuff when converting..to discuss

**Pull request merged**